src/components/ui/Button.tsx

```diff
@@ -18,6 +18,8 @@ const buttonVariants = cva(
         none: '',
         primary: 'bg-primary text-white enabled:hover:bg-primary-dark disabled:opacity-50',
         error: 'bg-danger text-white enabled:hover:bg-danger-dark disabled:opacity-50',
```

Comment:
Does naming `danger-outline` make sense to you? Please share with me if you have any better ideas

Reply:
I suggest aligning the red-filled and red-outlined buttons by using either words 'danger' or 'error'. 

I suggest naming them `danger` and `danger-outline` because, in our Figma design, it's called "danger". I think they were initially named with the word `error`, and that's why we named the color variant `error`, by the way.

<img width="400" alt="image" src="https://github.com/nishiki-tech/nishiki-frontend/assets/99148565/95594977-3c6d-4e6d-bef2-a0ee9713698e">

<img width="400" alt="image" src="https://github.com/nishiki-tech/nishiki-frontend/assets/99148565/bdc8a0df-3302-41ac-98e0-54b504d0112d">

Reply:
> I suggest naming them danger and danger-outline 

@kanta1207

What I meant here was to name them as `danger` and `danger-outline` instead of `error` and `error-outline`.